test/parallel/test-directory-import.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+{
+  assert.rejects(import('./'), /ERR_MODULE_NOT_FOUND/);
```

Comment:
Can we test that the error message here contains a `Did you mean` hint per the referenced PR for this?